client-src/elements/chromedash-drawer.js

```diff
@@ -0,0 +1,165 @@
+import {LitElement, html, css, nothing} from 'lit';
+import {SHARED_STYLES} from '../css/shared-css.js';
+
+
+export class ChromedashDrawer extends LitElement {
+  static get styles() {
+    return [
+      ...SHARED_STYLES,
+      css`
+        :host {
+          --md-gray-100-alpha: hsla(0, 0%, 0%, 0.12);
+          --md-gray-700-alpha: hsla(0, 0%, 0%, 0.62);
+          --md-blue-900: #01579b;
+
+          --nav-link-color: var(--md-gray-700-alpha);
+          --nav-link-font-size: 16px;
+          --nav-link-hover-background: var(--md-gray-100-alpha);
+          --nav-link-border: 2px solid transparent;
+          --nav-link-active-color: var(--md-blue-900);
+          --nav-link-active-border: 2px solid var(--nav-link-active-color);
+        }
+        nav {
+          display: flex;
+          flex: 1;
+          align-items: baseline;
+          user-select: none;
+          background: var(--card-background);
+          border-bottom: var(--card-border);
+          box-shadow: var(--card-box-shadow);
+          align-items: center;
+          margin: 0 var(--content-padding);
+          -webkit-font-smoothing: initial;
+        }
+        nav a {
+          text-decoration: none !important;
+          cursor: pointer;
+          font-size: var(--nav-link-font-size);
+          text-align: left;
+          padding: var(--content-padding-half) var(--content-padding);
+          color: var(--nav-link-color);
+          white-space: nowrap;
+          border-bottom: var(--nav-link-border);
+        }
+        nav a:hover {
+          color: black;
+          background: var(--nav-link-hover-background);
+        }
+        nav a.disabled {
+          opacity: 0.5;
+          pointer-events: none;
+        }
+        nav [active] {
+          color: var(--nav-link-active-color);
+          border-bottom: var(--nav-link-active-border);
+        }
+        nav [active] a {
+          color: var(--nav-link-active-color);
+        }
+        nav .nav-dropdown-container {
```

Comment:
Maybe we could just make the Stats item always open rather than collapsable.  We have enough space and it would simplify the code and the UX if it was just always open.

Reply:
Done